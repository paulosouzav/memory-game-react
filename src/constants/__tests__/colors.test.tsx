import { COLORS, TRANSPARENT_BLACK } from '../colors';

describe('COLORS', () => {
  it('should have expected values', () => {
    expect(COLORS).toMatchSnapshot();
  });
});

describe('TRANSPARENT_BLACK', () => {
  it('should have expected values', () => {
    expect(TRANSPARENT_BLACK).toMatchSnapshot();
  });
});
