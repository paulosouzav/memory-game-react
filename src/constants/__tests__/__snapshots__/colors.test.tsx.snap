// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COLORS should have expected values 1`] = `
Object {
  "black": "#000000",
  "blue": "#007bff",
  "cyan": "#17a2b8",
  "gray100": "#f8f9fa",
  "gray200": "#e9ecef",
  "gray300": "#dee2e6",
  "gray400": "#ced4da",
  "gray500": "#adb5bd",
  "gray600": "#6c757d",
  "gray700": "#495057",
  "gray800": "#343a40",
  "gray900": "#212529",
  "green": "#28a745",
  "indigo": "#6610f2",
  "orange": "#fd7e14",
  "pink": "#e83e8c",
  "purple": "#6f42c1",
  "red": "#dc3545",
  "teal": "#20c997",
  "transparent": "rgba(0,0,0,0)",
  "white": "#ffffff",
  "yellow": "#ffc107",
}
`;

exports[`TRANSPARENT_BLACK should have expected values 1`] = `
Object {
  "hard": "rgba(0,0,0,0.7)",
  "normal": "rgba(0,0,0,0.5)",
  "soft": "rgba(0,0,0,0.2)",
}
`;
